invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme.less

```diff
@@ -202,8 +202,8 @@ h6 {
 }
 
 .btn-warning {
```

Comment:
minor: we should be able to change this name:
- Semantic uses `button` instead of `btn`.
- It is not actually a "warning button". Could be "sign-up-button" class or something.

I'd say go ahead, merge and then open an issue to revist this in the future.

Reply:
I opened an issue https://github.com/inveniosoftware/invenio-app-rdm/issues/160